string action1
int32 obj_index_1
geometry_msgs/Point position1
string action2
int32 obj_index_2
geometry_msgs/Point position2
string speed
string angle
